or_slam:
  pure_localization: false

  # 仿真时无用
  use_sim: false
  
  odom_frame_id: "odom"
  base_frame_id: "base_link"
  global_frame_id: "map"
  laser_topic_name: "scan"
  trunk_topic_name: "trunk_obs"
  transform_tolerance: 0.1
  trunk_std_radius: 0.0668

  initial_pose_x: 0 #2.5
  initial_pose_y: 0 #8
  initial_pose_a: 0
  initial_cov_xx: 0.1
  initial_cov_yy: 0.1

  #地图输出位置
  slam_map_file_name: /home/cxn/myfile/OR_ws/orchardrover_ws/src/OrchardRover/or_making_map/slam_map.txt
  #地图读取位置
  read_map_file_name: /home/cxn/myfile/OR_ws/orchardrover_ws/src/OrchardRover/or_making_map/correct_map.txt

